distros/spring/src/main/java/com/connexta/transformation/service/TransformController.java

```diff
@@ -13,86 +13,86 @@
  */
 package com.connexta.transformation.service;
 
-import com.connexta.transformation.commons.api.TransformationManager;
+import com.connexta.transformation.commons.api.exceptions.TransformationException;
+import com.connexta.transformation.commons.api.status.Transformation;
+import com.connexta.transformation.commons.inmemory.InMemoryTransformationManager;
 import com.connexta.transformation.rest.models.TransformRequest;
-import com.connexta.transformation.rest.springboot.TransformApi;
-import com.google.common.annotations.VisibleForTesting;
+import com.connexta.transformation.rest.spring.TransformApiTransform;
 import com.google.common.base.Preconditions;
-import io.swagger.annotations.ApiParam;
+import java.net.URI;
+import java.net.URISyntaxException;
 import javax.validation.Valid;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.client.RestTemplate;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
-/**
- * Implementation of the Transformation RESTful service. This is the main entry point for all HTTP
- * requests.
- */
+/** Implementation of the Transformation RESTful POST/TRANSFORM service */
 @RestController
 @CrossOrigin(origins = "*")
-public class TransformController implements TransformApi {
+public class TransformController implements TransformApiTransform {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TransformController.class);
 
   private static final String ACCEPT_VERSION = "Accept-Version";
 
-  private final RestTemplate restTemplate;
-
-  private final String lookupServiceUrl;
-
-  public TransformationManager transformationManager;
+  private final InMemoryTransformationManager transformationManager;
```

Comment:
Do we want to use `TransformationManager` here instead of the `InMemoryTransformationManager`?

Reply:
I need to be able to use InMemoryTransformationManager.createTransform() that jeter's InMemoryTransformationManager PR added in

Reply:
fixed